sysfs/redis_view/app.go

```diff
@@ -52,7 +51,7 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
-		if(msg.Channel=="lights") {
+		if msg.Channel == "lights" {
```

Comment:
Brackets removal wasnt me (I happen to prefer the bracket notation) - VSCode strikes again